### DISCUSSION TITLE
Surgery system and a criticism of the current healing system

### DISCUSSION THREAD

**Discussion by Raiva666** [FakeFishGames/Barotrauma#7237] [Ideas and suggestions]
As it is, the medical system is really, really barebones. Yet, somehow, most players struggle to learn it.
That's because the game makes a big deal out of something that is actually extremely basic and easy to learn.
The healing interface can be overwhelming, with all the injury and status effect icons, and several different treatments that could be used.

But the truth is, they're all put under very few categories, where, to put it bluntly, in large, they're all the same, with different names. The same goes for medical supplies.

Burns: You've got 3rd degree burns all over?
Plastiseal sorts that out.

Blood loss: You've lost most of your blood because there are several wounds open in your body?
A blood pack will fix it.

Internal Injuries:
So you've got most of the bones in your body broken, you're puking out your organs and you've a hole through your chest?
Just have some morphine and you'll be fine.

As it is, the healing system is akin to putting the square, triangle, and circle toys into the correctly-shaped hole.
There's a variety of medical items that have no use, such as glue, which is simply a downgrade from fentanyl or morphine, of which addiction isn't (and in my opinion, shouldn't be) a serious threat.
In practice, to survive most injuries in most situations, players only need three items.

Even without high medic skill, the essential medical items are still highly effective.
With books and talents, about everyone can level up their medical skill, making a dedicated medic largely useless. This isn't exclusive for the medic, but the way every role can be replaced is a different issue.

The system is still very impressive with the shown variety of possible injuries and the amount of medical supplies and side-effects present.
My proposition is to make the healing system more complex yet rewarding by implementing actual surgery with pseudo-realistic side-effects and consequences that'll shake gameplay up and add variety.

Damage in general can be healed so as to not be more threatening, but there's only so much modern medicine can do for you: dependent on the gravity of an injury, your maximum health points will have a penalty until your body heals on its own (a process that could be sped-up with certain medicines), and dependent on the type of injury along with the seriousness, there will be penalties for certain actions.
Burns will make your skin more sensitive, making you suffer extra damage until they heal. Different types of gel will be required depending on whether you've suffered first and second, or third degree burns.
Open wounds cause blood loss, a higher risk of infection, and require stitching or cauterizing.
Opioids would be required so as to endure the potential pain of the procedure, and could be used as a temporary minor fix for injuries, for in case surgery isn't currently possible. Bandages and plastiseal would only be temporary solutions. Lost blood would take more time to be recovered, and too much of a lack of blood would make your character more susceptible to nausea and several of the afflictions.
Injecting far too much blood would lead to high blood pressure, causing shortness of breath and nausea, and potential death if not treated.
For gunshot wounds, bullets and broken pieces of bone may have to be removed and medication against lead poisoning would be required.
Certain major wounds can't be healed completely, either permanently or only done so at high prices by specialized outpost medics, in similar vein to Reaper's Tax.
Depending on the type, using too many opium drugs in a short amount of time could lead to a fast beating heart, which could be a temporary reward as faster speed in several activities, but that would eventually lead to an overdose if not treated.
Broken bones require splints. In the case of broken leg bones, deal penalty to movement / swimming speed. Broken arms cause slower repair speeds and prevents you from picking up certain heavy or two-handed objects.
Inclusion of alien medical technology could be made to be hard to acquire but used as a better, faster treatment of injuries, with possible side-effects.

The goal of this is for a system that will incentive different playstyles and outcomes, a higher variety of combinations and possibilities with serious penalties that will punish less-than-careful players, while being intuitive and highly rewarding.

**Comment by HeatedWafflez** [comment]
[https://www.youtube.com/watch?v=evthRoKoE1o&ab_channel=Swiw](url)
barotrauma medical system

**Reply by MaaiG** [reply]
> [https://www.youtube.com/watch?v=evthRoKoE1o&ab_channel=Swiw](url)
> barotrauma medical system

Nice!
[Corrected your link!](https://www.youtube.com/watch?v=evthRoKoE1o[](url))

> As it is, the medical system is really, really barebones.

Nice!
- However, IMHO, needs to consider that it may be necessary to reduce the risk of various serious damage, so that the gameplay does not become a permanent treatment of the pricked finger or conveyor of crew members to the neurosurgeon.
It is possible that some minor injuries would affect the movement of the character, his vision, his adequacy, but could also heal on their own.
- Thus, the work of a physician becomes longer, interesting and complicated, but not too frequent. To keep the player’s time as a medic at an acceptable level. To develop specialization, but not for the amount of labor duties.
- Revision of medication consumption. For example: when blood bags are consumed like oxygen in a cylinder... ???
(When you need a lot of medicaments, you either: try carry a bunch of it around, or medicaments must doesn’t end so fast.)

**Reply by Raiva666** [reply]
I completely agree with you, but I believe not _much_ work would need to be done in case of decreasing chance of serious damage. I believe players would adapt and form ways to avoid damage more often. It'll probably make performance-enhancing drugs more highly utilized.
I do believe the actual surgery shouldn't take very long. Certain actions such as stitching could be done faster depending on medical skill, from 25 to 10 seconds, for example.

**Comment by StruggleDuck** [comment]
To be honest, I do agree that the medical system has its flaws, but I do not believe that the solution to the problem is to make it so that medics have to carry around a whole hospitals worth of tools and supplies, and make it next to impossible for your average joe to even attempt using the medical system. A better solution in my opinion would be to just re work some of the lesser used items, to make them stronger or have a more specific use, and then nerf the general items like morphine, fentanyl, and bandages. Even just by making morphine and fentanyl heal less, and making bandages heal much slower, it would put more emphasis on having more specialized medical items, while still not making the medical system require you to take a whole university degree to even understand it.

**Reply by Raiva666** [reply]
You wouldn't need a degree or actual medical knowledge to understand it. The game should assist you, as it already does with "suitable treatments". I believe making a more complicated system would make players not only want, but need to learn it. Assessment of the most likely possible injuries caused in the current mission would lead to fast medical work, and that's something players would acquire with experience (not the talent-tree kind).
Medics would still carry about as much as they already do; of course you can't surgeries on someone underwater, so only take items that should give them temporary fix-ups, such as opiates and bandages. It should last enough to complete an already half-way done nest mission and swim back to the submarine to give them proper treatment.

**Comment by ubertpendragon** [comment]
The medical system has been simplified to the extreme with the latest update. 
It's simple and functional, but also shallow

Some medical reworks have been suggested to make the system more complex
https://github.com/Regalis11/Barotrauma/discussions/5220
